lib/bolt/transport/remote.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'bolt/task/remote'
+require 'bolt/transport/base'
+
+module Bolt
+  module Transport
+    class Remote < Base
+      def self.options
+        # TODO: We should accept arbitrary options here
```

Comment:
Arbitrary as in configured? Is that something that's ticketed or should be ticketed?

Reply:
no we can't really meaningfully restrict it in the general case. This still needs to be ticketed.

Reply:
I may need to do this sooner to support the slack task